src/serialization/toSerializableFormat.js

```diff
@@ -32,15 +32,18 @@ function saveStatus(originalState, serializableState) {
  * @returns {*}
  */
 function transformSubstate(originalSubState) {
-  if (_.isPlainObject(originalSubState)) {
-    // eslint-disable-next-line  no-use-before-define
-    return toSerializableFormat(originalSubState);
-  } else if (_.isArray(originalSubState) && originalSubState[STATUS]) {
+  if (_.isArray(originalSubState)) {
+    if (!originalSubState[STATUS]) {
```

Comment:
I need ordinary serialization if it-s ordianry array so this second if is break condition before running arrayToObject logic.